disnake/app_commands.py

```diff
@@ -206,6 +206,15 @@ class Option:
         The minimum value permitted.
     max_value: Union[:class:`int`, :class:`float`]
         The maximum value permitted.
+    min_length: Optional[:class:`str`]
+        The minimum length for this option if the type of this Option is :attr:`OptionType.string`.
```

Comment:
The descriptions of `min_length`/`max_length` are somewhat inconsistent with 3 different ones being used in different places, it would be great if they used one common description like `The minimum length for this option, if it is a string option.`, for example.

Reply:
https://github.com/DisnakeDev/disnake/pull/593/commits/0eaab3da5bed267dfa64a2b1ca122c455e25ed51